PyU4V/provisioning.py

```diff
@@ -1674,14 +1679,17 @@ def create_non_empty_storage_group(
         :param vol_name: name to give to the volume, optional -- str
         :param snapshot_policy_ids: list of one or more snapshot policies
                                     to associate with storage group -- list
+        :param enable_mobility_id: enables unique volume WWN not tied to array
+                                   serial number -- bool
+
         :returns: storage group details -- dict
         """
         return self.create_storage_group(
             srp_id, storage_group_id, service_level, workload,
             do_disable_compression=disable_compression,
             num_vols=num_vols, vol_size=vol_size, cap_unit=cap_unit,
-            _async=_async, vol_name=vol_name,
-            snapshot_policy_ids=snapshot_policy_ids)
+            _async=_async, enable_mobility_id=enable_mobility_id,
```

Comment:
Again, totally nit picking.  Is there a particular reason that enable_mobility_id is not at the end?  It just for maintainability.   its easier to see a new parameter tagged on to the end rather than stuck in the middle.

Reply:
will change,